connectivity/connectivity-demos-test/pom.xml

```diff
@@ -93,6 +93,16 @@
       <version>4.2.0</version>
       <scope>test</scope>
     </dependency>
+
+    <!-- fix this version. because axon ivy engine is using commons.text 1.11.0 which
+        requires commons.lang3 3.13.0. but this maven dependency hierarchy includes an older
+        version -->
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.13.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
would be better to define 'exclusions' on the dependency that has this transient dependency.
it seems like there are already other 'clashing' dependencies outline on them.